snap/snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: git-machete
+version: 'v2.12.9'
```

Comment:
Would be nice if could be derived automatically from git_machete/__init__.py ... I'll play wiith that